README.md

```diff
@@ -54,6 +54,11 @@ shipyard set org {org-name}
 shipyard get org
 ```
 
+### Set a Shipyard token
+```bash
+shipyard set token
```

Comment:
```suggestion
shipyard set token {token}
```

Reply:
While we do support this flow, we should encourage users to not provide the token as an argument because it's a sensitive value that will show up in history. Without arguments, the command will ask for the token interactively, which is more secure.

Reply:
ah duh, forgot we do that, will update the readme docs as well.